src/components/views/right_panel/HeaderButton.tsx

```diff
@@ -45,9 +45,9 @@ export default class HeaderButton extends React.Component<IProps> {
         const { isHighlighted, isUnread = false, onClick, name, title, ...props } = this.props;
 
         const classes = classNames({
-            "mx_RoomHeader_button": true,
-            "mx_RoomHeader_button--highlight": isHighlighted,
-            "mx_RoomHeader_button--unread": isUnread,
+            "mx_LegacyRoomHeader_button": true,
```

Comment:
This implies this should be LegacyHeaderButton

Reply:
Sounds good, will update the files name. You're right.

Reply:
Actually, these classes should not be existing in this file in the first place. I will be keeping this file with the current name and will not spend time refactoring code we're looking to remove.